src/main/resources/static/simple.soy

```diff
@@ -8,7 +8,7 @@
     {call aui.form.textField}
         {param id: 'dotnetFormatParams' /}
 		{param labelContent: 'Additional parameters' /}
-		{param value:$config ? $config['dotnetFormatParams'] : null /}
+		{param value:$config ? $config['dotnetFormatParams'] : '--check' /}
```

Comment:
We could make it so `--check` shows up as a default value on UI:
```suggestion
		{param value:$config ? $config['dotnetFormatParams'] ? $config['dotnetFormatParams'] : '--check' : null /}
```
It's a bit ugly but does work.